rust_src/src/frames.rs

```diff
@@ -81,6 +110,32 @@ impl LispFrameRef {
         unsafe { fset_selected_window(self.as_mut(), window.to_raw()) }
     }
 
+    #[inline]
+    pub fn terminal(&self) -> Terminal {
+        unsafe { Terminal::new(fget_terminal(self.as_ptr())) }
+    }
+
+    /// Return `DisplayInfo` about `self` if `self` is a TTY or MSDOS raw display,
```

Comment:
For what it's worth, we don't support MS-DOS. The comment is harmless when we still have mentions of DOS in our structs though.